.github/workflows/release.yml

```diff
@@ -0,0 +1,68 @@
+# Releases the latest version of Optic on NPM/Brew/Debian
+# Assumes the version has been updated
+# Triggers on all changes to the release branch
+
+name: Release
+
+env:
+  HOMEBREW_REPO: opticdev/homebrew-optic
+  BUCKET_NAME: optic-deb-packages
```

Comment:
moving over this question from slack, do we want `optic-deb-packages` or `optic-packages` and then we'd specify a `--prefix` for deb-s3 below that would result in bucket path that is like, `optic-packages/deb/filename`

Reply:
changed to `optic-packages`!